Source/WebCore/platform/encryptedmedia/CDMInstanceSession.h

```diff
@@ -79,7 +82,7 @@ class CDMInstanceSession : public RefCounted<CDMInstanceSession> {
     };
 
     using LicenseCallback = CompletionHandler<void(Ref<SharedBuffer>&& message, const String& sessionId, bool needsIndividualization, SuccessValue succeeded)>;
-    virtual void requestLicense(LicenseType, const AtomString& initDataType, Ref<SharedBuffer>&& initData, LicenseCallback&&) = 0;
+    virtual void requestLicense(LicenseType, KeyGroupingStrategy, const AtomString& initDataType, Ref<SharedBuffer>&& initData, LicenseCallback&&) = 0;
```

Comment:
Looks like this change is causing problems for other ports. Maybe have two `virtual` methods, and have the new one just assume `CDMKeyGroupingStrategy::Platform` by default?